### PR TITLE
Fix yaml format for dev tests

### PR DESCRIPTION
**Pull request by fxmarty**
The dev yaml were badly formatted.

Reference: https://github.com/huggingface/optimum/actions/runs/3984098460

I hope it's now good.

### CODE REVIEW

File: .github/generate_dev_tests.py [hunk 1]
```diff
@@ -34,12 +34,23 @@
...
+        workflox_yml = workflox_yml.replace("run:", "run: |\n       ")
+
+        workflox_yml = "\n".join([ll.rstrip() for ll in workflox_yml.splitlines() if ll.strip()])
+
+        outfile.seek(0)
+        outfile.write(workflox_yml)
+        outfile.truncate()
+
 
     with open(Path("workflows", new_name), "r+") as outfile:
```
Comment (regisss) [inline .github/generate_dev_tests.py:55]: This code block could be merged with the one right above

Comment (fxmarty) [reply .github/generate_dev_tests.py]: Thanks, fixed!

File: .github/generate_dev_tests.py [hunk 2]
```diff
@@ -34,17 +34,21 @@
...
             workflox_yml,
             outfile,
-            default_style="|",
             default_flow_style=False,
             allow_unicode=True,
             width=float("inf"),
             sort_keys=False,
         )
-
+
```
Comment (regisss) [inline .github/generate_dev_tests.py:42]: Nit: extra tabulation here

**Review by regisss — APPROVED** [review]
LGTM!

**Pull request merged**